posts/templates/posts/feed.html

```diff
@@ -4,60 +4,68 @@
+                        {% if post.author == user %}
+                            {% if post.resume %}
+                                <a href="{% url 'resume-update' post.pk %}" class="btn btn-outline-secondary btn-light btn-sm"><div>Edit post</div></a>
                             {% endif %}
-                          {% endif %}
-                          <a href="{% url 'comment-create' post.pk %}"><button type="button" class="btn btn-secondary">Add comment</button></a>
-                          <hr>
-                        </div>
-                            {% for comment in post.comment_set.all %}
-                              <div class="comment-data">
-                                <img class="rounded-circle article-img" src="{{ comment.author.profile.profile_pic.url }}">
-                                <a class="mr-2" href="#">{{ comment.author }}</a>
-                                <hr class="comment-hr">
-                                <p class="comment-content">{{ comment.comment_text }}</p>
-                              </div>
-                            {% endfor %}
+                            <a href="{% url 'post-delete' post.pk %}" class="btn btn-outline-danger btn-light btn-sm" ><div>Delete post</div></a>
+                        {% endif %}
                     </div>
+                 </div>
+             </div>
+        </div>
+        <p class="article-content">{{ post.description }}</p>
+        <div class="file-and-comments">
+            {% if post.resume %}
+                <iframe src="{{ post.resume.resume_file.url }}" style="width: 560px;height: 780px ;border: none;"></iframe>
+            {% endif %}
+            <div class="comments-block">
+                <div class="buttons">
+                  {% if post.resume %}
+                  {% include 'posts/rating_view.html' %}    
+                    {% if post.author != user %}
+                      <div>
+                          <a href="{% url 'create-a-rate' post.pk %}"><button type="button" class="btn btn-secondary">Rate ResuMe</button></a>
+                          <hr>
+                      </div>
+                    {% endif %}
+                  {% endif %}
+                  <a href="{% url 'comment-create' post.pk %}"><button type="button" class="btn btn-secondary">Add comment</button></a>
+                  <hr>
                 </div>
+                    {% for comment in post.comment_set.all %}
+                      <div class="comment-data">
+                        <img class="rounded-circle article-img" src="{{ comment.author.profile.profile_pic.url }}">
+                        <a class="mr-2" href="#">{{ comment.author }}</a>
+                        <hr class="comment-hr">
+                        <p class="comment-content">{{ comment.comment_text }}</p>
+                      </div>
+                    {% endfor %}
             </div>
-        </article>
-    {% endfor %}
+        </div>
+    </div>
+</article>
```

Comment:
I think the indentation should be with 2 spaces according to the article tag opening. Also, how do you decide when to use 2 spaces or a tab?
```suggestion
  </article>
```

Reply:
fixed, now all the indentations are with 2 tabs (like all the HTML files we have)